tests/k8s/1-005_validate-custom-argocd-namespace/02-create-app.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: validate-custom-argocd
+  namespace: test-1-8-custom
+spec:
+  project: default
+  source:
+    repoURL: https://github.com/redhat-developer/gitops-operator
```

Comment:
Relying on something in the gitops-operator repo might be a little fragile; a change in this other repo could break this test.  Maybe we should create the equivalent app here in the argocd-operator repo.